### ISSUE TITLE
Are you using Angular MDC? Tell us about your use

### ISSUE THREAD

**Issue by trimox** [trimox/angular-mdc-web#1870]
Thought it'd be fun for the community to showcase their use of Angular Material Components. What are you using the components for? If it's public, feel free to share a link!

**Comment by zunamo** [comment]
I am using it currently for [Jetset Weather](https://jetsetweather.com)

Thanks for this library and the support you provide!

**Comment by trimox** [comment]
@zunamo Great job! That's a beautiful site! :gem:

**Comment by vguleaev** [comment]
I tried to use it for personal project, but because of big lack of some components (Datepicker, Autocomplete), I refused to use it in favor of Angular Material :(

**Comment by trimox** [comment]
@vguleaev No worries! I do have to say MDC has a **much** requested items in the pipeline; DataTable https://github.com/material-components/material-components-web/issues/4591). They'll eventually implement Date Picker and Autocomplete, but until luckily we have libraries to fill gaps 😃

**Comment by vguleaev** [comment]
@trimox Yes! Exactly! I remembered the reason was not only Date Picker but lack of tables! That is sooo much core component, that's why I refused to use it  :) At first I wanted to use table from Angular Material but setup 2 almost the same frameworks were disaster for me

**Comment by vguleaev** [comment]
> I am using it currently for [Jetset Weather](https://jetsetweather.com)
> 
> Thanks for this library and the support you provide!

@zunamo this website is awesome

**Comment by zunamo** [comment]
@vguleaev - Thanks a lot!

**Comment by phutaneVinayak** [comment]
Hi @trimox ,
angular-mdc-web great Lib. Thanks for your constant support and response.
 I am using in our web application [www.easyroads.in](easyroads.in) for mobile version.

**Comment by BrianThorne** [comment]
I am using Angular MDC for a web project I'm working on. I'll post a link when it's presentable. Thanks for providing this library, it's perfect for my needs.

**Comment by spiotr12** [comment]
HI
I'm creating library-like app using mdc web components in Angular. I'm also creating a wrapper library that puts this library components into directives. <not sure if that's right approach but I did not want to create tones of components>

I would loved to use data-table component soon, it is bloking me a bit.

**Comment by dj8923** [comment]
Please comment to resolve my doubts - 
1 - Can i use both together **angular-mdc-web** and **angular material v8.1.2** ? 
2 - Or Should i avoid the combination of both ?
3 - Will the look and feel will be constant if I use both of them ?

**Comment by vekunz** [comment]
@dj8923 We use both in our project. Mainly we use the MDC, but we add it with AngularMaterial. Mostly the components don't need to work together so it is no problem. But one time I had to combine the AngularMaterial Datepicker with a MDC TextField. It was a bit of a hack, but it works 😂

**Comment by arabbani** [comment]
I was thinking about using this library for my next project. But I can see it lacks some basic components like datepicker, tooltip, and many others

**Comment by mohammadrafigh** [comment]
We are using Angular MDC on [*thisApp](https://my.thisapp.io) project which aims to spread crypto-currencies love. *thisApp also runs on Android and iOS so Angular MDC did a great job here! :+1: 
for more info you can visit main [website](https://thisapp.io).
